WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewController.swift

```diff
@@ -127,6 +134,43 @@ private extension CardReaderSettingsConnectedViewController {
         tableView.reloadData()
     }
 
+    func configureUpdateView() {
+        guard viewModel != nil, let viewModelUpdateInProgress = viewModel?.readerUpdateInProgress else {
```

Comment:
I would suggest this:
`guard let viewModel = viewModel, let viewModelUpdateInProgress = viewModel.readerUpdateInProgress, updateInProgress != viewModelUpdateInProgress else `
What seems a bit odd to me is not having two different guard clauses, which is okay, but the negative check in the first guard, that does not read very familiar in Swift.

Reply:
Thanks! Simplified in https://github.com/woocommerce/woocommerce-ios/pull/4867/commits/a117c90bb6b68e167a4f2f15dcc7027b7049af7b